github/src/main/kotlin/gropius/sync/github/model/IssueInfo.kt

```diff
@@ -21,8 +22,8 @@ import java.time.OffsetDateTime
 data class IssueInfo(
     @Indexed(unique = true)
     var githubId: String,
-    @Indexed(unique = true)
-    var neo4jId: String, val dirty: Boolean, var lastAccess: OffsetDateTime?, val imsProject: String
+    val url: URI,
```

Comment:
are you sure you formatted this file?
this looks a bit strange to me

Reply:
maybe. will not touch it in hopes of less conflicts

Reply:
🙄

Reply:
I am grasping at any straws in hope of not snapping because of these unneeded complications